person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initilize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_service
+    of_age || parent_permission == true
+  end
```

Comment:
- [ ] There seems to be an issue with your `can_use_services` method. When the age of the person is less than 18, an error is returned as a result of this `parent_permission == true` because it is not defined (see image below). I would suggest you use this instance variable instead and refactor the method to be more concise, like so:

  ```
  def can_use_service
    of_age || @parent_permission
  end
  ```

  ![nas-khalil-services](https://user-images.githubusercontent.com/36542986/156020523-14906cda-19b9-45e9-aefd-6eb0070b5208.jpg)